cli/deno_dir.rs

```diff
@@ -27,6 +27,18 @@ use std::str;
 use url;
 use url::Url;
 
+fn normalize_path(path: &Path) -> PathBuf {
```

Comment:
Does this even make sense in when using `Path` and `PathBuf` instead of `String`? 

I think you should try removing this function entirely.

Reply:
I believe it does... PTAL at [this comment](https://github.com/denoland/deno/pull/2572#issuecomment-504941862). 

I'd prefer to normalize all paths to use forwards slashes everywhere in our codebase.